pkg/server/sandbox_remove.go

```diff
@@ -56,6 +55,11 @@ func (c *criContainerdService) RemovePodSandbox(ctx context.Context, r *runtime.
 		return nil, fmt.Errorf("sandbox container %q is not fully stopped", id)
 	}
 
+	// Return error if sandbox network namespace is not closed yet.
+	if sandbox.NetNS != nil && !sandbox.NetNS.Closed() {
```

Comment:
@Random-Liu qq: so what is the recovery here? How is the network namespace eventually removed and closed ? And if its not closed , what is the behavior if the sandbox is restored ?